### ISSUE TITLE
Bug in xpbinview

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#733]
A user reported an edge case triggering an exception in xpbinview. 

The offending code is binning/polarization.py, where we setup the grid for plotting the polarization cubes

```
        # Setup the radial grid for the polarization angle. In order to have a
        # sensible default, here, we judge based on the plot side and try and
        # setup a sensible regular grid.
        if pd_grid is None:
            if side <= 0.05:
                step = 0.01
            elif side <= 0.1:
                step = 0.02
            elif side <= 0.25:
                step = 0.05
            else:
                step = 0.1
            pd_grid = numpy.arange(step, side, step)
```

More specifically, when `side < step`, `pd_grid` is empty and this breaks the code downstream.

**Comment by lucabaldini** [comment]
Link to the pull request: https://github.com/lucabaldini/ixpeobssim/pull/734

**Comment by lucabaldini** [comment]
Shipped in version 31.0.3

**Comment by SeshadriMajumder** [comment]
Thanks, Luca! It is working now. The issues are gone!
